src/services/expense.service.js

```diff
@@ -0,0 +1,70 @@
+const { getId } = require('../utils/getId');
+
+let expenses = [];
+
+const initialize = () => {
+  expenses = [];
+};
+
+const getExpenses = (userId, categories, from, to) => {
+  return expenses.filter(
+    (item) =>
+      (!userId || item.userId === Number(userId)) &&
+      (!categories || item.category === categories) &&
+      (!from || !to || (item.spentAt >= from && item.spentAt <= to)),
+  );
+};
+
+const create = (
+  userId,
+  spentAt = new Date().toISOString(),
+  title,
+  amount = 1,
+  category,
+  note,
```

Comment:
```suggestion
  userId,
  title,
  category,
  note,
  amount = 1,
  spentAt = new Date().toISOString(),
```

let's move non required params to the end of list, will be more readable